src/scripts/items/MulchItem.ts

```diff
@@ -17,7 +17,7 @@ class MulchItem extends Item {
     }
 }
 
-ItemList['Boost_Mulch']   = new MulchItem(MulchType.Boost_Mulch, 50, 'Boost Mulch');
-ItemList['Rich_Mulch']  = new MulchItem(MulchType.Rich_Mulch, 100, 'Rich Mulch');
-ItemList['Surprise_Mulch']  = new MulchItem(MulchType.Surprise_Mulch, 150, 'Surprise Mulch');
-ItemList['Amaze_Mulch'] = new MulchItem(MulchType.Amaze_Mulch, 200, 'Amaze Mulch');
+ItemList['Boost_Mulch']   = new MulchItem(MulchType.Boost_Mulch, 50, 'Boost Mulch', 'Increases Berry growth rate.');
+ItemList['Rich_Mulch']  = new MulchItem(MulchType.Rich_Mulch, 100, 'Rich Mulch', 'Increases Berry harvest rate.');
+ItemList['Surprise_Mulch']  = new MulchItem(MulchType.Surprise_Mulch, 150, 'Surprise Mulch', 'Increases Berry mutation rate.');
+ItemList['Amaze_Mulch'] = new MulchItem(MulchType.Amaze_Mulch, 200, 'Amaze Mulch', 'Increases all Berry effects.');
```

Comment:
Not related to this area of the code,
But it's probably also worth adding description to the shop Oak Items.

Reply:
Yeah... I wanted to, but that would require duplicating the strings (or at least storing them somewhere separate). At this point in the code we don't have access to the OakItems yet, so we can't just call App.game.oakItems.itemList[SquirtBottle].description. The string would have to be stored somewhere static and used in both locations instead.

Reply:
Or couldn't we just make oak shop items automatically fetch the oak items description?

Reply:
I'm not sure how I feel about adding some sort of race condition on initialization. Is there a way we can ensure that items will always load first?

Reply:
Only way I can think of right now is making them static then referencing from the item 🤔 😩

Reply:
I guess it'll have to do until we do some kind of I18N refactor so multiple locations can share the same string.

Reply:
> until we do some kind of I18N refactor

I honestly don't think that's going to happen any time soon, if ever..

Reply:
I've just added a default description for all shop Oak Items.